### ISSUE TITLE
[Input] Review Accessibility

### ISSUE THREAD

**Issue by diogofscmariano** [lumada-design/hv-uikit-react#248]
### Description: 
Review the input component accessibility topics, which should contain:
- interactible through keyboard 
- browser reader and its order
- explanation on how to use clear button and validations
- label and input should be connected
- validation text 
- info icon is not focusable
- disabled should be added with its aria property
- zooming capabilities

This component documentation needs to state his WAI level. 

Questions:
- how do we expose this information to who wants to use our component?
- how to reach the x or show password icon, if reachable.
- how to inform the user about the validation

**Estimation: 8** 

**Acceptance Criteria:** 
- https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_customize&tags=forms&levels=aa%2Caaa&technologies=smil%2Cpdf%2Cflash%2Csl#labels-or-instructions
- Add tests to validate the accessibility guidelines where it makes sense
